concourse_ci/README.md

```diff
@@ -0,0 +1,33 @@
+## Overview
+
+Capture metrics emitted by concourse-ci.
```

Comment:
This is a bit short. Can we add a bit more content, for example what type of metrics we collect? (c.f. Travis CI and CircleCI integrations)

Reply:
Let's be consistent with the product name: "Concourse CI", and not ~concourse-ci~, ~Concourse-CI~ ...

Reply:
👍